api/presenter/launch.go

```diff
@@ -0,0 +1,47 @@
+package presenter
+
+import (
+	"encoding/json"
+	"github.com/soyantonio-w/academy-go-q12021/entity"
+)
+
+type LaunchPresenter struct {
```

Comment:
Remember to ident tags column

Reply:
Thank you, I have just run the go format 😄